<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_8</class>
 <widget class="QDialog" name="fct_comp_8">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reservation</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choisir un sp√©ctacle </string>
   </property>
  </widget>
  <widget class="QComboBox" name="NoSpec_fct_combo_8_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dateRep_fct_combo_8_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NoSpec</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DateRep</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NoRang_fct_combo_8_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NoRang</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NoPlace</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NoPlace_fct_combo_8_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choisir une place</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choisir un dossier</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NoDos_fct_combo_8_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NoDos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reserver</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_fct_comp_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>751</width>
     <height>461</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>NoSpec</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DateRep</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>noRang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>noPlace</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DateEmTick          </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>libelleCat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>noDos</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>libelleCat</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fct_combo_8_12">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>NoSpec_fct_combo_8_1</zorder>
  <zorder>dateRep_fct_combo_8_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>NoRang_fct_combo_8_3</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>NoPlace_fct_combo_8_4</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>NoDos_fct_combo_8_5</zorder>
  <zorder>label_10</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_11</zorder>
  <zorder>table_fct_comp_8</zorder>
  <zorder>label_2</zorder>
  <zorder>fct_combo_8_12</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>NoSpec_fct_combo_8_1</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8</receiver>
   <slot>refreshDateRep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NoRang_fct_combo_8_3</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8</receiver>
   <slot>refreshNoPlace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>Reserver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshDateRep()</slot>
  <slot>refreshNoPlace()</slot>
  <slot>refreshResult()</slot>
  <slot>Reserver()</slot>
 </slots>
</ui>
